views/artist-search-results.hbs

```diff
@@ -0,0 +1,15 @@
+
+{{#each items}}
+<section>
+    <p>Name: {{this.name}}</p>
```

Comment:
if you wanna show image : 
some of them dont have any image, so you can check if image exist , show it, it could be like this :
`{{#if this.images.0.url}}
          <img
            src={{this.images.0.url}}
            alt={{this.name}}
            class="card-img-top"
          />
          {{/if}}`